src/App.js

```diff
@@ -37,7 +38,7 @@ const App = () => (
                 icon={ProductIcon}
             />
             <Resource name={'categories'} list={CategoryList} icon={CategoryIcon} />
-            <Resource name={'customers'} list={CustomerList} icon={UserIcon} />
+            <Resource name={'customers'} icon={UserIcon} list={CustomerList} edit={CustomerEdit} />
```

Comment:
Just a note, we usually have an `index` file in each resource directory where we re-export their components as a single object. It allows a shorter resource declaration:

```js
// in customers/index.js
import UserIcon from '@material-ui/icons/Group';

import { CustomerList } from './CustomerList';
import { CustomerEdit } from './CustomerEdit';

export default {
    list: CustomerList,
    edit: CustomerEdit,
    icon: UserIcon
}

// in App.js
import customers from './customers';

<Resource name={'customers'} {...customers} />
```

Reply:
ok thanks. I will do it